### DISCUSSION TITLE
[Plausible 2.0] Can't invite user anymore

### DISCUSSION THREAD

**Discussion by Striffly** [plausible/analytics#3224] [Self-Hosted Support] [answered]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

When I try to invite someone to view analytics, user receive invitation email.
However, when clicking on the link inside to create account, user is redirected to login page. No form is shown.

### Expected behavior

See the account creation form

### Environment

```markdown
### Plasuible

- Version : 2.0
- Running on : Docker
- Migrated from 1.x to 2.0

### User

- OS: Arch Linux
- Browser: Chromium
- Browser Version: 115.0.5790.110
```

**Comment by ruslandoga** [comment, marked as answer]
👋 @Striffly 

In `v2.0.0` the default for [`DISABLE_REGISTRATION`](https://plausible.io/docs/self-hosting-configuration#server) has been changed from `false` to `true`. This is probably the reason for the form not being shown. You can set that env var to `invite_only` to make it work.

**Reply by pineapples5972** [reply]
Can I ask How does the `invite_only` works in registration. Is it for allowing people to see the stat of particular site based on inviteonly or its for creating account in global instance via sending a "invite only link" for registration?

**Reply by ruslandoga** [reply]
Answered in #4057
